apis/nucleus/src/components/listbox/components/ListBoxSearch.jsx

```diff
@@ -7,46 +7,72 @@ import InstanceContext from '../../../contexts/InstanceContext';
 
 const TREE_PATH = '/qListObjectDef';
 
-export default function ListBoxSearch({ model, keyboard, dense = false, visible = true }) {
+export default function ListBoxSearch({ selections, model, keyboard, dense = false, visible = true }) {
   const { translator } = useContext(InstanceContext);
   const [value, setValue] = useState('');
-  const [isSearching, setIsSearching] = useState(false);
   const theme = useTheme();
 
+  const cancel = () => selections.isActive() && selections.cancel();
+
   const abortSearch = async () => {
-    await model.abortListObjectSearch(TREE_PATH);
-    setIsSearching(false);
+    if (!selections.isModal()) {
+      return;
+    }
```

Comment:
This only works if the listbox uses model selection.
That is currently true but we may want to change that in the future.
(More specific I think that the use in analytic creation we may want to try using listboxes without modal selection)

Reply:
Ok, good to know. For this fix I don't think we need to consider these anticipated changes though, since we probably have this dependency elsewhere in our applications that needs to be removed when that day comes. `isActive()` is not a direct equivalent of `isModal` unfortunately, don't know if we have any good proxy for this, do you know?